pages/help/help_domain-management.md

```diff
@@ -79,6 +79,32 @@ After renewal, the domain is registered for another year. We don’t offer an au
 4. Update as needed.
 5. Click “Save.”
 
+<div class="usa-accordion usa-accordion--bordered">
+  <h4 class="usa-accordion__heading">
+    <button
+      type="button"
+      class="usa-accordion__button"
+      aria-expanded="false"
+      aria-controls="enterprise-mailing-address"
+    >
+      Update your organization mailing address in enterprise mode
+    </button>
+  </h4>
+  <div id="enterprise-mailing-address" class="usa-accordion__content usa-prose">
+
+In enterprise mode, only users in an “org admin” role can make updates to the organization’s mailing address.
+
+
+1. Sign in to the [.gov registrar](https://manage.get.gov){target="_blank"} using your Login.gov account.
+2. Click your organization name in the top navigation bar to access the “Organization overview” page.
```

Comment:
If users belong to more than one organization (portfolio), they'll need to select it first before seeing the org name in the top-nav.